### ISSUE TITLE
The parent/child relationship doesn't work as expected.

### ISSUE THREAD

**Issue by LeeHanYeong** [adamghill/django-unicorn#485]
Even if you change the parent's value in the child component, it doesn't change in the template.
The parent component's children property value at the time of completion of rendering is empty at other times.
I attach an example project repository. After pip install -r requirements.txt, you can check it with runserver.
https://github.com/LeeHanYeong/unicorn-issue

**Parent**
```python
class SampleView(UnicornView):
    sample_value = "default"

    def sample(self):
        self.sample_value = "sample"

    def rendered(self, html):
        print(self.children)

    def updated(self, name, value):
        # updated not called
        print("updated", name, value)

    def set_sample(self, value):
        self.sample_value = value
        print(f"parent.set_sample({value})")

        # self.children is Empty
        # (All children present at the time of rendering disappear)
        print(self.children)
```

**Child1**
```python
class ChildView(UnicornView):
    def child_click(self):
        print("child_click")
        self.parent.sample_value = "child"
        # Parent's value doesn't change in template
        print(self.parent.sample_value)
```

**Child2**
```python
class Child2View(UnicornView):
    def child2_click(self):
        print("child2_click")
        self.parent.set_sample("child2")
        # Parent's value doesn't change in template
        print(self.parent.sample_value)
```

**Comment by amserra** [comment]
Having the same issue here.

**Comment by adamghill** [comment]
I think this should be fixed by 0.58.0 of `Unicorn` which drastically improves child components so I'm going to close it. If it is still a problem feel free to re-open this. Thank you!
